src/tensorlake/function_executor/service.py

```diff
@@ -72,7 +72,7 @@ def initialize(
             graph_version=request.graph_version,
             fn=request.function_name,
         )
-        
+
```

Comment:
Despite this being a nothing change, lets remove this file change from this PR since it seems to have just been a mistake.